packages/lesswrong/components/comments/CommentForm.tsx

```diff
@@ -63,8 +64,10 @@ const formStyles = defineStyles('CommentForm', (theme: ThemeType) => ({
   fieldWrapper: {
     marginTop: theme.spacing.unit * 2,
     marginBottom: theme.spacing.unit * 2,
-    background: theme.palette.editor.bannerAdBackground,
-    color: theme.palette.text.bannerAdOverlay,
+    [isIfAnyoneBuildsItFrontPage]: {
```

Comment:
This should not be wrapped

Reply:
Moderate disagree.

Reply:
I don't feel strongly, so let's wrap it for now